packages/app-project/public/locales/en/screens.json

```diff
@@ -55,16 +55,6 @@
         "title": "There was an error in the classifier :("
       }
     },
-    "FinishedForTheDay": {
-      "buttons": {
-        "stats": "See the stats"
-      },
-      "text": "Your answers are saved for the research team while you're working. See the project stats and return to the {{projectName}} home page.",
-      "title": "Finished for the day?",
-      "ProjectImage": {
-        "alt": "Image for {{projectName}}"
-      }
-    },
```

Comment:
This component is deleted for good, so its translation keys are deleted throughout.